src/wasm/wasm-stack.cpp

```diff
@@ -114,13 +114,18 @@ void BinaryInstWriter::visitLocalSet(LocalSet* curr) {
 }
 
 void BinaryInstWriter::visitGlobalGet(GlobalGet* curr) {
-  o << int8_t(BinaryConsts::GlobalGet)
-    << U32LEB(parent.getGlobalIndex(curr->name));
+  Index index = parent.getGlobalIndex(curr->name);
```

Comment:
likewise here, maybe a comment saying that for tuples we split them up etc? and in Set below.